app/src/main/java/tv/vizbee/demo/retrofit/ApiInterface.kt

```diff
@@ -0,0 +1,27 @@
+package tv.vizbee.demo.retrofit
+
+import retrofit2.Call
+import retrofit2.http.Body
+import retrofit2.http.Header
+import retrofit2.http.POST
+import retrofit2.http.Path
+import tv.vizbee.demo.model.LoginRequest
+import tv.vizbee.demo.model.LoginResponse
+import tv.vizbee.demo.model.RegCodeStatusRequest
+
+interface ApiInterface {
```

Comment:
for naming of the classes or packages, refer to any other repo such as fox or GitHub famous repos. I think we don’t name them after retrofit. We may name them by the purpose. Please check once.

Reply:
Done